### PR TITLE
net-dns/knot: version bump

### PR DESCRIPTION
**Pull request by nemunaire**

### CODE REVIEW

File: net-dns/knot/knot-2.5.4.ebuild [hunk 1]
```diff
@@ -0,0 +1,101 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit systemd user
+
+DESCRIPTION="High-performance authoritative-only DNS server"
+HOMEPAGE="http://www.knot-dns.cz/"
```
Comment (a17r) [inline net-dns/knot/knot-2.5.4.ebuild]: you could switch to https here.

File: net-dns/knot/knot-2.5.4.ebuild [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+	caps? ( >=sys-libs/libcap-ng-0.6.4 )
+	dnstap? (
+		dev-libs/fstrm
+		dev-libs/protobuf-c
+	)
+	idn? (
+		!libidn2? ( net-dns/libidn )
+		libidn2? ( >=net-dns/libidn2-2.0.0 )
+	)
+	dev-libs/libedit
```
Comment (a17r) [inline net-dns/knot/knot-2.5.4.ebuild]: just a suggestion and not particularly necessary for this bump: it is good practice to keep dependencies sorted and optional dependencies below non-optional.

**Comment by nemunaire** [comment]
Fixed!

```diff
--- a/net-dns/knot/knot-2.5.4.ebuild
+++ b/net-dns/knot/knot-2.5.4.ebuild
@@ -6,7 +6,7 @@ EAPI=6
 inherit systemd user
 
 DESCRIPTION="High-performance authoritative-only DNS server"
-HOMEPAGE="http://www.knot-dns.cz/"
+HOMEPAGE="https://www.knot-dns.cz/"
 SRC_URI="https://secure.nic.cz/files/knot-dns/${P/_/-}.tar.xz"
 
 LICENSE="GPL-3"
@@ -27,10 +27,11 @@ KNOT_MODULES=(
 IUSE="doc caps +fastparser idn libidn2 systemd +utils ${KNOT_MODULES[@]}"
 
 RDEPEND="
-       >=net-libs/gnutls-3.3:=
        >=dev-db/lmdb-0.9.15
-       dev-python/lmdb
+       dev-libs/libedit
        >=dev-libs/userspace-rcu-0.5.4
+       dev-python/lmdb
+       >=net-libs/gnutls-3.3:=
        caps? ( >=sys-libs/libcap-ng-0.6.4 )
        dnstap? (
                dev-libs/fstrm
@@ -40,7 +41,6 @@ RDEPEND="
                !libidn2? ( net-dns/libidn )
                libidn2? ( >=net-dns/libidn2-2.0.0 )
        )
-       dev-libs/libedit
        systemd? ( >=sys-apps/systemd-229 )
 "
 DEPEND="${RDEPEND}
```

**Comment by a17r** [comment]
Thanks!

**Pull request closed without merging**